detections/application/ollama_possible_api_endpoint_scan_reconnaissance.yml

```
@@ -0,0 +1,59 @@
+name: Ollama Possible API Endpoint Scan Reconnaissance
+id: ad3f352a-0347-48ee-86b9-670b5025a548
+version: 1
+date: '2025-10-05'
+author: Rod Soto
+status: production
+type: TTP
+description: Detects API reconnaissance and endpoint scanning activity against Ollama servers by identifying sources probing multiple API endpoints within short timeframes, particularly when using HEAD requests or accessing diverse endpoint paths, which indicates systematic enumeration to map the API surface, discover hidden endpoints, or identify vulnerabilities before launching targeted attacks.
+data_source:
+- Ollama Server
+search: '`ollama_server` "GIN" 
```

To avoid matching sub texts like `engine` and what not
```suggestion
search: '`ollama_server` "[GIN]" 
```